docs/FAQ.rst

```diff
@@ -0,0 +1,276 @@
+A: We cannot commit to any timeline to add any particular operator, still we are open to external
+contributions. On top of that every user can extend DALI on his own without modifying its code
+base, please check `this page <../examples/custom_operations/index.html>`_ for more details.
+
+Q: How should I know if I should use a CPU or GPU operator variant?
+###################################################################
+A: It depends on the particular use case. I would start by checking GPU  utilization to see if
+throwing more work on the GPU won't slow things down, or just place everything on the CPU first
+and then gradually move things to the GPU and see if that helps. You can check
+`our GTC22 talk <https://www.nvidia.com/gtc/session-catalog/#/session/1636559250287001p4DG>`_
+or `the blog post <https://developer.nvidia.com/blog/case-study-resnet50-dali/>`_ to see how
+to do it step by step.
+
+Q: How can I provide a custom data source/reading pattern to DALI?
+##################################################################
+A: You can define any custom data loading pattern using python and
+`external source operator <../examples/general/data_loading/external_input.html>`_. To make it
+faster please use `its parallel capability <../examples/general/data_loading/parallel_external_source.html>`_.
+
+Q: Does DALI provide any profiling capabilities?
+################################################
+A: DALI doesn't provide any built-in profiling capabilities, still it utilizes NVTX ranges
+and has a dedicated domain (so it is easy to find in the profile) to show its operations. So you can
+capture the profile using `NVIDIA Nsight Systems <https://developer.nvidia.com/nsight-systems>`_
+or any Deep Learning profile that also supports NVTX markers.
+
+Q: Does DALI support multi GPU/node training?
+#############################################
+A: Yes, DALI supports data-parallel and distributed data-parallel strategies. Its shards data into
+non-overlapping pieces using the number of shards (world size) and shard id (global rank), and
+uses device id to identify the GPU used in the particular node (local rank).
+
+Q: How to report an issue/RFE or get help with DALI usage?
+##########################################################
+A: DALI is an open-source project hosted on GitHub, you can ask questions and report issues
+using `this link <https://github.com/NVIDIA/DALI/pulls>`_ directly.
+
+Q: Does DALI accelerate only processing or also the data loading itself?
+########################################################################
+A: DALI mostly focuses on processing acceleration, as in most cases the input data is compressed
+(audio, video, or images) and the input data is relatively small compared to the raw decoded output.
+Still, there are cases, like NumPy array files, where data is not compressed and in such a case
+DALI can use `GPUDirect Storage <https://developer.nvidia.com/gpudirect-storage>`_ to bypass CPU
```

Comment:
This sounds like it's done for any uncompressed file format, which is not the case.

Reply:
Reworked